internal/rvtools/parser.go

```diff
@@ -132,10 +132,11 @@ func ParseRVTools(ctx context.Context, rvtoolsContent []byte, opaValidator *opa.
 			VmsPerCluster:         ExtractVmsPerCluster(vInfoRows),
 		}
 		vcenterInventory := service.CreateBasicInventory(&vms, infraData)
+		datastoreIDToType := buildDatastoreIDToTypeMapFromRVTools(datastoreRows)
```

Comment:
We already have the "buildDatastoreMapping" function that does similar loop
(line 42)
I wonder if we can use the same function to create this map as well